docs/attributes-registry/mcp.md

```diff
@@ -0,0 +1,53 @@
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/attribute_namespace.md.j2 -->
+
+# MCP
+
+## MCP Attributes
+
+[Model Context Protocol (MCP)](https://github.com/modelcontextprotocol/specification) attributes
+
+| Attribute | Type | Description | Examples | Stability |
+|---|---|---|---|---|
+| <a id="mcp-method-name" href="#mcp-method-name">`mcp.method.name`</a> | string | The request or notification method name. | `notifications/cancelled`; `initialize`; `notifications/initialized` | ![Development](https://img.shields.io/badge/-development-blue) |
+| <a id="mcp-prompt-name" href="#mcp-prompt-name">`mcp.prompt.name`</a> | string | The name of the prompt or prompt template provided in the request or response. | `analyze-code` | ![Development](https://img.shields.io/badge/-development-blue) |
+| <a id="mcp-request-argument" href="#mcp-request-argument">`mcp.request.argument.<key>`</a> | string | Additional arguments passed to the request within `params` object. `<key>` being the normalized argument name name (lowercase), the value being the argument value. [1] | `mcp.request.argument.location=Seattle, WA`; `mcp.request.argument.a=42` | ![Development](https://img.shields.io/badge/-development-blue) |
+| <a id="mcp-request-id" href="#mcp-request-id">`mcp.request.id`</a> | string | This is a unique identifier for the request. | `42` | ![Development](https://img.shields.io/badge/-development-blue) |
+| <a id="mcp-resource-uri" href="#mcp-resource-uri">`mcp.resource.uri`</a> | string | The value of the resource uri. [2] | `postgres://database/customers/schema`; `file:///home/user/documents/report.pdf` | ![Development](https://img.shields.io/badge/-development-blue) |
+| <a id="mcp-session-id" href="#mcp-session-id">`mcp.session.id`</a> | string | Identifies MCP session. | `191c4850af6c49e08843a3f6c80e5046` | ![Development](https://img.shields.io/badge/-development-blue) |
+| <a id="mcp-tool-name" href="#mcp-tool-name">`mcp.tool.name`</a> | string | The name of the tool provided in the request. | `get-weather`; `execute_command` | ![Development](https://img.shields.io/badge/-development-blue) |
+
+**[1] `mcp.request.argument`:** Instrumentations SHOULD require an explicit configuration of which arguments are to be captured. Including all request arguments can be a security risk - explicit configuration helps avoid leaking sensitive information.
+Argument values SHOULD be recorded as JSON strings.
+
+**[2] `mcp.resource.uri`:** This is a URI of the resource when the method is is `resources/read`, `resources/subscribe`, `resources/unsubscribe` or `notifications/resources/updated`.
```

Comment:
URI should probably be the URI template, and then provide the values for the template fields using mcp.request.argument.<key>

Reply:
it's literally the URI parameter in the message https://github.com/modelcontextprotocol/modelcontextprotocol/blob/45466e9cccdb49e68746f6c3a005d67a8aa0a16d/schema/2025-03-26/schema.ts#L379 or https://github.com/modelcontextprotocol/modelcontextprotocol/blob/45466e9cccdb49e68746f6c3a005d67a8aa0a16d/schema/2025-03-26/schema.ts#L408, I can only report what's there. 

Also these requests don't have arguments.